lib/clock.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'clockwork'
+require File.expand_path('../../config/boot',        __FILE__)
+require File.expand_path('../../config/environment', __FILE__)
+
+include Clockwork
+
+handler do |job|
```

Comment:
Could be an idea to include the second argument `time` and print this out to the Rails logger too, that way if we have any problems there might be more of a chance of figuring out when it started. (see the first example in the section https://github.com/Rykian/clockwork#quickstart).

Reply:
I could, but we will get this I believe from the timestamp from the log line in any case.